src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
> Try testing that on TestSwarm then. :-) 

We can't right now, right now we do only ua-testing, which is bad substitution. We can check this with e2e-tests. You might not remember this, but those ua tests were created as temp solution.

> Complicating that logic would be confusing.

I don't think it would complicate stuff, the opposite - you have function that checks if fix should be applied, that is it, nothing else, nothing more - all encapsulated. Now this is all united, dependant, tma and all that, which is bad.

you could do
 `if (isNeeded()) { "there you go" }`

instead of doing
 `if ("maybe-browser-has-this" || "or-maybe-that" && "but-if-doesn't-then") { "wtf was that?!" }`